util/envDynawo.sh

```diff
@@ -1359,7 +1365,13 @@ deploy_dynawo() {
   mkdir -p extraLibs/LIBARCHIVE/lib/
   mkdir -p extraLibs/LIBZIP/lib
   mkdir -p extraLibs/LIBXML/lib
-  cp -P $SUNDIALS_INSTALL_DIR/lib64/*.* 3rdParty/sundials/lib/
+  if [ -d "$SUNDIALS_INSTALL_DIR/lib64" ]; then
+    cp -P $SUNDIALS_INSTALL_DIR/lib64/*.* 3rdParty/sundials/lib/
```

Comment:
Can be done without if: cp -P $SUNDIALS_INSTALL_DIR/lib*/*.* 3rdParty/sundials/lib/